src/main/java/io/th0rgal/oraxen/api/OraxenItemsAPI.java

```diff
@@ -0,0 +1,32 @@
+package io.th0rgal.oraxen.api;
+
+import io.th0rgal.oraxen.items.OraxenItems;
+import org.bukkit.inventory.ItemStack;
+
+/**
+ * Class for Oraxen custom items
+ */
+public class OraxenItemsAPI {
+    /**
+     * Return Oraxen item ID of the provided {@link ItemStack}.
+     * <br> May return null if the provided {@link ItemStack} is not an Oraxen item.
+     *
+     * @param itemStack {@link ItemStack} to get the ID
+     * @return Possibly-null Oraxen item ID
+     */
+    public static String getOraxenItemID(ItemStack itemStack) {
+        String itemID = OraxenItems.getIdByItem(itemStack);
+        if (itemID == null) return null;
+        return itemID;
+    }
+
+    /**
+     * Return ItemStack of the provided Oraxen item ID.
+     *
+     * @param itemID Oraxen item ID
+     * @return {@link ItemStack}
+     */
+    public static ItemStack getOraxenItemStack(String itemID) {
+        return OraxenItems.getItemById(itemID).build();
+    }
+}
```

Comment:
Not sure what the point is when you have access to OraxenItems
ur just making it have extra steps now

Reply:
my idea is that it can be accessed in a more friendlier way with clear description and like that
also lazy method in a single package make it more accessable ig

Reply:
All this does tho is add another class with the API tag at the end
It really doesnt do much. Would say for backwards compatibility, add methods to OraxenItems and add similar classes

This is just extra steps with different methodnames